Cargo.toml

```diff
@@ -1,33 +1,64 @@
 [package]
-name = "rimage"
-version = "0.11.0-next.2"
-edition = "2021"
-description = "Optimize images natively with best-in-class codecs"
-license = "MIT OR Apache-2.0"
-readme = "README.md"
-authors = ["Vladyslav Vladinov <vladinov.dev@gmail.com>"]
-keywords = ["image", "compression", "encoder"]
-categories = ["compression", "multimedia::images"]
-repository = "https://github.com/SalOne22/rimage"
+name          = "rimage"
+version       = "0.11.0-next.2"
+edition       = "2021"
+description   = "Optimize images natively with best-in-class codecs"
+license       = "MIT OR Apache-2.0"
+readme        = "README.md"
+authors       = ["Vladyslav Vladinov <vladinov.dev@gmail.com>"]
+keywords      = ["image", "compression", "encoder"]
+categories    = ["compression", "multimedia::images"]
+repository    = "https://github.com/SalOne22/rimage"
 documentation = "https://docs.rs/rimage"
-homepage = "https://lib.rs/rimage"
-include = ["/README.md", "/Cargo.toml", "/src/**/*.rs"]
+homepage      = "https://lib.rs/rimage"
+include       = ["/README.md", "/Cargo.toml", "/src/**/*.rs"]
+build         = "build.rs"
+
+    [package.metadata.winres]
```

Comment:
remove here indentation

Reply:
I've tried it but rust-analyzer continuing add indentations when formatting...

I'll modify it manually.